pkg/lib/operation/template/local/test/operation.go

```diff
@@ -1,20 +1,213 @@
 package test
 
 import (
+	"context"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
 
+	cliDeps "github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/dialog"
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/options"
+	"github.com/keboola/keboola-as-code/internal/pkg/cli/prompt/nop"
+	"github.com/keboola/keboola-as-code/internal/pkg/env"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem/aferofs"
+	fixtures "github.com/keboola/keboola-as-code/internal/pkg/fixtures/local"
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/project"
 	"github.com/keboola/keboola-as-code/internal/pkg/template"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/input"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testfs"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper/storageenvmock"
+	useTemplate "github.com/keboola/keboola-as-code/pkg/lib/operation/project/local/template/use"
+	loadState "github.com/keboola/keboola-as-code/pkg/lib/operation/state/load"
 )
 
 type Options struct {
 	Path string
 }
 
 type dependencies interface {
+	Ctx() context.Context
 	Logger() log.Logger
 }
 
-func Run(_ *template.Template, _ dependencies) (err error) {
-	return fmt.Errorf(`not implemented`)
+// getATestProject takes first project from TEST_KBC_PROJECTS env var.
+//@TODO from github.com/keboola/go-utils/pkg/testproject/testproject.go:155.
+func getATestProject() (string, string, string) {
```

Comment:
S tymto planujes este nieco robit?
Na prvy pohlad som neprisiel na to, kde je problem.

Reply:
Jo vlastně, o tom jsme se bavili po tom co jsi se vrátil z dovolené. Že je třeba upravit https://github.com/keboola/go-utils/blob/main/pkg/testproject/testproject.go aby fungoval i bez `testing.T`. Musí se tam nějak nahradit ta cleanup funkce (https://github.com/keboola/go-utils/blob/be395f3a33c1a01a57d7b42f060663369880caad/pkg/testproject/testproject.go#L114-L116) a to úplně nevím jak na to i když jsi mi to myslím popisoval. 😅

Reply:
Ale to zamykání tam asi ani potřeba není.(?) Mohlo by stačit jen to rozparsování (tj. zavolání `initProjects()` a potom vrácení `projects` proměnný), upravím aby se to dalo použít zvenku.

Reply:
No to zamykanie je tam nato, aby si mohol paralelne spustit viac testov. A hned ako sa nejaky projekt uvolni, tak spustit dalsi test. Vzhladom na pocet sablon (20) sa to moze hodit, ale nemal by s tym byt problem (?)

Reply:
Tak už hotovo (https://github.com/keboola/keboola-as-code/pull/752/commits/75d35335b893e9e6c7f8c4c5f902a25f022f2e1c a https://github.com/keboola/keboola-as-code/pull/752/commits/67b19e18da7b4849e66eec5f3f2b008c003bf842)